client/src/App.js

```diff
@@ -1,15 +1,23 @@
 import React from 'react';
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
-import { 
-        ApolloClient,
-        ApolloProvider,
-        InMemoryCache,
-        createHttpLink,
+import {
+  ApolloClient,
+  ApolloProvider,
+  InMemoryCache,
+  createHttpLink,
 } from '@apollo/client';
 import { setContext } from '@apollo/client/link/context';
-import SearchBooks from './pages/SearchBooks';
-import SavedBooks from './pages/SavedBooks';
+import LandingPage from './pages/LadingPage';
```

Comment:
Just a missing 'n' typo.

```suggestion
import LandingPage from './pages/LandingPage';
```